kafka-client/src/main/java/org/sunbird/kafka/client/KafkaClient.java

```diff
@@ -0,0 +1,69 @@
+package org.sunbird.kafka.client;
+
+import java.util.Properties;
+import org.apache.kafka.clients.consumer.Consumer;
+import org.apache.kafka.clients.consumer.ConsumerConfig;
+import org.apache.kafka.clients.consumer.KafkaConsumer;
+import org.apache.kafka.clients.producer.KafkaProducer;
+import org.apache.kafka.clients.producer.Producer;
+import org.apache.kafka.common.serialization.LongDeserializer;
+import org.apache.kafka.common.serialization.StringDeserializer;
+import org.sunbird.common.models.util.LoggerEnum;
+import org.sunbird.common.models.util.ProjectLogger;
+
+/**
+ * @author Mahesh Kumar Gangula This class will create Kafka consumer and producer object. and
+ *     provide to caller. For each call it will create new consumer and producer object.
+ */
+public class KafkaClient {
+
+  private static Producer<Long, String> createProducer(String bootstrapServers, String clientId) {
```

Comment:
Expose these as public methods and rename createPropertiesObj to createProperties.